tslint.json

```diff
@@ -74,6 +74,7 @@
     "no-string-literal": false,
     "no-switch-case-fall-through": true,
     "no-var-requires": false,
+    "prefer-for-of": false,
```

Comment:
Reason for this is because in a unit test, an HTML Query List does not implement the iterator type, requiring us to use a regular js for loop